src/components/Quiz.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import Answers from './Answers';
+import { publish } from './Event';
+import QuestionAnswer from './QuestionAnswer';
+import Score from './Score';
+import TimerContainer from './TimerContainer';
+
+function Quiz() {
+  const [questionNumber, setQuestionNumber] = useState(1);
+  const [questions, setQuestions] = useState([]);
+  const nextQuestion = (question) => {
+    if (questionNumber < 21) {
```

Comment:
These values from should be coming from constant

Reply:
Done